Mathlib/Algebra/Homology/ShortComplex/HomologicalComplex.lean

```diff
@@ -0,0 +1,67 @@
+/-
+Copyright (c) 2023 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+
+import Mathlib.Algebra.Homology.HomologicalComplex
+import Mathlib.Algebra.Homology.ShortComplex.Homology
+
+/-! The short complexes attached to homological complexes
+
+In this file, we define a functor
+`shortComplexFunctor C c i : HomologicalComplex C c ⥤ ShortComplex C`.
+By definition, the image of an homological complex `K` by this functor
+is the short complex `K.X (c.prev i) ⟶ K.X i ⟶ K.X (c.next i)`.
+
+When the homology refactor is completed (TODO @joelriou), the homology
+of an homological complex `K` in degree `i` shall be the homology
```

Comment:
I'm not a native speaker, but this semms to be more consistent with existing usages (for example in `Homology.lean`).
```suggestion
By definition, the image of a homological complex `K` by this functor
is the short complex `K.X (c.prev i) ⟶ K.X i ⟶ K.X (c.next i)`.

When the homology refactor is completed (TODO @joelriou), the homology
of a homological complex `K` in degree `i` shall be the homology
```
(There are two more occurrences in docstrings below)